npc/quests/quests_gonryun.txt

```diff
@@ -3760,8 +3759,8 @@ gonryun,237,226,3	script	Han Ran Jiao#gon	4_M_TWMIDMAN,{
 		mes "here! Where did you get it!?";
 		next;
 		mes "[Han Ran Jiao]";
-		mes "I dropped that knife from being";
-		mes "careless. It belongs to one of my";
+		mes "I carelessly dropped it by accident.";
```

Comment:
Carelessly dropped by accident.
Something doesn't sound good here.
Maybe "I carelessly dropped the knife?"